azure-pipelines.yml

```diff
@@ -199,15 +213,11 @@ stages:
                 $signAssemblies = [System.Convert]::ToBoolean("$(IS_RELEASE_BRANCH)") # the variable is a string, we need a boolean
                 Package-NetFrameworkScanner -SignAssemblies $signAssemblies
                 Package-NetScanner -SignAssemblies $signAssemblies
-              pwsh: true
 
-          - task: NuGetCommand@2
+          - powershell: |
+                nuget help | Select-String "NuGet Version"
```

Comment:
This looks like a debug output. Without it, transferring this task back to `NuGetCommand@2` should be possible.

Reply:
See [this comment](https://github.com/SonarSource/sonar-scanner-msbuild/pull/1933#discussion_r1639843424).
And [this documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/nuget-command-v2?view=azure-pipelines#remarks).
It was done to force the use of a newer Nuget version than 4.1.0 (which is the default for this task and can't be configured through parameters).

Reply:
Reverted it. Seems to work. Keeping the old version.